.github/workflows/deploy.yml

```diff
@@ -53,8 +54,17 @@ jobs:
         if: matrix.os == 'windows-latest'
         run: pip install -r requirements-windows.txt
 
-      - name: 📦 Packaging
-        run: bash ./package_antares_web.sh
+      - name: Initialize installer environment
+        run: |
+          python -m venv .installer
+          source .installer/bin/activate
+          python -m pip install -r requirements.txt
+        working-directory: scripts
+
+      - name: Build installer
+        run: |
+          bash ./package_antares_installer.sh
```

Comment:
I think you don't call anymore the main packaging script which builds antares desktop  :) 
It still needs to be called after the installer build, since it creates the final zip that will contain everything